src/sessions/softphone-session-handler.ts

```diff
@@ -539,11 +531,22 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
       participant = await this.fetchUserParticipantFromConversationId(pendingSession.conversationId);
     }
 
+    if (participant.purpose === 'user') {
+      return this._rejectUcCall(pendingSession.conversationId, participant.id);
+    }
+
     return this.patchPhoneCall(pendingSession.conversationId, participant.id, {
       state: CommunicationStates.disconnected
     });
   }
 
+  _rejectUcCall (conversationId: string, participantId: string): Promise<any> {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9630720)
```suggestion
  _rejectUcCall (conversationId: string, participantId: string): Promise<unknown> {
```